logstash-core/api/lib/app/root.rb

```diff
@@ -6,7 +6,7 @@ class Root < BaseApp
 
     get "/" do
       content = { "name" => "Logstash API",
-                  "version" => { "number" => "0.1.0" },
+                  "version" => { "number" => LOGSTASH_CORE_VERSION },
```

Comment:
I think it should be `LOGSTASH_VERSION` instead of `LOGSTASH_CORE_VERSION` the latter is the version of the gem that we use and its _might_ be out of sync with the release version.